speedcopy/__init__.py

```diff
@@ -220,6 +220,63 @@ def copyfile(src, dst, follow_symlinks=True):
         return dst
 
 else:
+    # Windows
+    # Initialize once on import for performance
+    kernel32 = ctypes.WinDLL('kernel32',
+                             use_last_error=True,
+                             use_errno=True)
```

Comment:
I know too little about `ctypes` to know whether keeping a handle to a DLL like this would be of any trouble or not.